src/commands/android/subcommands/uninstall/avd.ts

```diff
@@ -0,0 +1,53 @@
+import colors from 'ansi-colors';
+import inquirer from 'inquirer';
+
+import Logger from '../../../../logger';
+import {Platform} from '../../interfaces';
+import {getBinaryLocation} from '../../utils/common';
+import {execBinarySync, execBinaryAsync} from '../../utils/sdk';
+import {showMissingBinaryHelp} from '../common';
+
+export async function deleteAvd(sdkRoot: string, platform: Platform): Promise<boolean> {
+  try {
+    const avdmanagerLocation = getBinaryLocation(sdkRoot, platform, 'avdmanager', true);
+    if (!avdmanagerLocation) {
+      showMissingBinaryHelp('avdmanager');
+
+      return false;
+    }
+
+    const installedAvds = execBinarySync(avdmanagerLocation, 'avdmanager', platform, 'list avd -c');
+    if (!installedAvds) {
+      Logger.log(`${colors.yellow('Failed to fetch installed AVDs.')} Please try again.`);
```

Comment:
We should also provide a command here that the user can run to list all the installed AVDs (probably using the dot command?).

Also, do you think it'd be worth adding a `list` subcommand to the tool, to list various things like `list devices` to list all the connected devices and `list avd` to list all the avds?

Reply:
Sounds like a good idea. We can add a subcommand for listing devices.